### PR TITLE
Fixes instr order for default translation

### PR DESCRIPTION
**Pull request by edgararuiz**
Fixes https://github.com/tidyverse/dplyr/issues/3397

Notes:

- Tested it against an Oracle and SQLite connections
- Re-tested the MS SQL custom translation and found that it needed no changes

### CODE REVIEW

**Comment by edgararuiz** [comment]
ptal @hadley

File: R/translate-sql-base.r [hunk 1]
```diff
@@ -167,7 +167,7 @@ base_scalar <- sql_translator(
                       )},
   str_detect      = function(string, pattern){
```
Comment (hadley) [inline R/translate-sql-base.r]: Can you please add a comment here pointing to the documentation fo INSTR?

Comment (edgararuiz) [reply R/translate-sql-base.r]: I have links to string functions for the different databases at the beginning of the section were we added the `stringr` functions: https://github.com/tidyverse/dbplyr/blob/81419cab7d831e5a9d7e4279a4f893fa5750e8a1/R/translate-sql-base.r#L154-L160 
Does this suffice?

Comment (hadley) [reply R/translate-sql-base.r]: Oh yeah, that's sufficient.

**Comment by hadley** [comment]
Thanks!

**Comment by leungi** [comment]
Thanks @edgararuiz and @hadley!

**Pull request merged**